### PR TITLE
fix(blog): Blog Back Button

### PR DESCRIPTION
**Pull request by sharad-s**
## Why?

We want to update the back button styles in the blog

## How?

- Updated css
- Created Button variant

## Tickets?

- [AS-227](https://linear.app/fleekxyz/issue/AS-227/update-design-for-back-button-on-blog-post)

## Contribution checklist?

- [X] The commit messages are detailed
- [X] The `build` command runs locally
- [X] Assets or static content are linked and stored in the project
- [X] Document filename is named after the slug
- [X] You've reviewed spelling using a grammar checker
- [X] For documentation, guides or references, you've tested the commands and steps
- [X] You've done enough research before writing

## Security checklist?

- [x] Sensitive data has been identified and is being protected properly
- [x] Injection has been prevented (parameterized queries, no eval or system calls)
- [x] The Components are escaping output (to prevent XSS)

## References?

Optionally, provide references such as links

## Preview?
Old:
image

New:

image

image

### CODE REVIEW

**Review by victorgaard — CHANGES_REQUESTED** [review]
Hey @sharad-s the styling for `Go back` button is correct, but I'd suggest creating a `GoBackButton` or a different component for now, as `ButtonGray` is used in different places of the website. Not ideal I know, but the website buttons need some refactor. 

Some side effects of changing `ButtonGray`

image

---

image

**Comment by sharad-s** [comment]
Thanks! Good point, I’ll go ahead and refactor

File: src/pages/blog/[...slug].astro [hunk 1]
```diff
@@ -35,7 +35,13 @@ const date = formatDate({ date: entry.data.date });
 >
   <article class="blog">
     <div class="mb-24 w-fit" id="goBackButton">
-      <ButtonGray>Go back</ButtonGray>
+      <ButtonGray variant="text">
+        <img
+          src="/svg/caret-left.svg"
+          class="caret mr-1 border-0 !border-none !outline-none invert filter"
```
Comment (gaarf) [inline src/pages/blog/[...slug].astro:41]: `w-[2rem]` ?

**Review by victorgaard — APPROVED** [review]
Looking great, thanks! 🚀

**Pull request merged**